Src/tbconfig.cpp

```diff
@@ -53,6 +55,7 @@ void TBConfig::setValue(const char *key, const char *value) {
     else if (strcmp(key, "minLongPressMS") == 0) minLongPressMS = atoi(value);
     else if (strcmp(key, "qcTestState") == 0) qcTestState = atoi(value);
     else if (strcmp(key, "initState") == 0) initState = atoi(value);
+    else if (strcmp(key, "VOLUME_STEP") == 0)   maxVolumeStep = atoi(value);
```

Comment:
In "CreateForV2" the value is written as lower case. This should match, right?

Reply:
Sorry it was an oversight. Fixed it